tests/utils/test_display.py

```diff
@@ -0,0 +1,23 @@
+import pygfx as gfx
+import pytest
+from wgpu.gui.offscreen import WgpuCanvas
+
+
+def test_repeated_show():
+    pytest.xfail("Offscreen canvas can currently not be closed.")
+
+    cube = gfx.Mesh(
+        gfx.box_geometry(200, 200, 200),
+        gfx.MeshPhongMaterial(color="#336699"),
+    )
+
+    # always use offscreen canvas
+    # @almarklein do we have a fixture for that?
```

Comment:
We have the env var that makes the `auto` backend select the offscreen backend. But no special fixtures.

Reply:
That also only works if it's set prior to importing, so it's pretty hard to use if you're not launching a subprocess.

Reply:
Good point.

Reply:
Hm, in that case I would say we leave this test as is. Would you like me to open a new issue for writing a fixture that forces to use an offscreen canvas during unit tests?

Reply:
That would need to be an issue at wgpu-py. It would mean we need to make a function so we can resolve after import time.